web/src/components/ProjectScreen.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import styled from "@emotion/styled";
+import useTasks from "../graphql/useTasks";
+import TaskContent from "./TaskContent";
+import TaskDetailModal from "./TaskDetailModal";
+import EditPermissionsModal from "./EditPermissionsModal";
+import Card from "./Card";
+import Button from "@leafygreen-ui/button";
+import TextInput from "@leafygreen-ui/text-input";
+import { uiColors } from "@leafygreen-ui/palette";
+
+export default function ProjectScreen({
+  currentProject,
+  isEditingPermissions,
+  setIsEditingPermissions,
+}) {
+  return (
+    <Container>
+      {currentProject && <TaskList currentProject={currentProject} />}
+      <EditPermissionsModal
+        project={currentProject}
+        isEditingPermissions={isEditingPermissions}
+        setIsEditingPermissions={setIsEditingPermissions}
+      />
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  grid-area: main;
+  background: ${uiColors.gray.light2};
+`;
+
+function useDraftTask({ addTask }) {
+  const [draftTask, setDraftTask] = React.useState(null);
+  const createDraftTask = () => {
+    setDraftTask({ name: "" });
+  };
+  const setDraftTaskName = (name) => {
+    setDraftTask({ name });
+  };
+  const submitDraftTask = async () => {
+    await addTask(draftTask);
+    setDraftTask(null);
+  };
+  return { draftTask, createDraftTask, setDraftTaskName, submitDraftTask };
+}
+
+function TaskList({ currentProject }) {
+  const { tasks, addTask } = useTasks(currentProject);
+  const getTaskById = (id) => tasks.find((task) => task._id === id);
+  const [selectedTaskId, setSelectedTaskId] = React.useState(null);
+  const selectedTask = getTaskById(selectedTaskId);
+
+  const { draftTask, createDraftTask, setDraftTaskName, submitDraftTask } = useDraftTask({ addTask })
+
+  return (
+    <>
+      <List>
+        {tasks.map((task) => (
+          <ListItem key={task._id}>
+            <Card onClick={() => setSelectedTaskId(task._id)}>
+              <TaskContent task={task} />
+            </Card>
+          </ListItem>
+        ))}
+        {draftTask && (
+          <ListItem>
+            <Card>
+              <TextInput
+                type="text"
```

Comment:
```suggestion
                type="text"
                aria-labelledby="task description"
```

will get rid of the warning: "For screen-reader accessibility, label or aria-labelledby must be provided to IconButton."
<img width="479" alt="Screen Shot 2020-09-17 at 1 41 17 PM" src="https://user-images.githubusercontent.com/52428905/93507299-7ad4f080-f8eb-11ea-93b0-98e017d4f92d.png">

Reply:
Yup good call! Fixed one of these already but didn't get the one in the edit permissions modal.